package.json

```diff
@@ -100,25 +100,24 @@
     "elasticsearch": "^14.2.1",
     "elasticsearch-browser": "^14.2.1",
     "encode-uri-query": "1.0.0",
-    "even-better": "7.0.2",
     "execa": "^0.10.0",
     "expiry-js": "0.1.7",
     "extract-text-webpack-plugin": "3.0.1",
     "file-loader": "1.1.4",
     "font-awesome": "4.4.0",
     "glob": "^7.1.2",
     "glob-all": "^3.1.0",
+    "good": "github:elastic/good#17.5.3-kibana1",
     "good-squeeze": "2.1.0",
-    "h2o2": "5.1.1",
+    "h2o2": "8.1.2",
     "h2o2-latest": "npm:h2o2@8.1.2",
     "handlebars": "4.0.5",
-    "hapi": "14.2.0",
-    "hapi-latest": "npm:hapi@17.5.0",
+    "hapi": "17.5.3",
```

Comment:
Let's go ahead and use carets for the version matching here.

Reply:
SGTM